### PR TITLE
feat: Development of simple webpage for online toolkit

### PR DESCRIPTION
**Pull request by Sotatek-QuangDo**

### CODE REVIEW

**Review by edwin082 — APPROVED** [review]
Nice Work! And I just added a copy button.

File: src/routes.tsx [hunk 1]
```diff
@@ -285,6 +286,19 @@ const routes: RouteType[] = [
...
+  {
+    name: 'Converters',
+    path: '/converter',
+    items: [
+      {
+        path: '/klaytnUnitConverters',
+        name: 'Klaytn Unit Converter',
+        component: KlaytnUnitConverter,
+        description:
+          'The Peb converter from Klaytn is a simple and easy-to-use tool for converting between Peb, ston, and Klay. The page housing this tool provides a comprehensive explanation of these different units and their relationship to the expenditure of Gas in the Klaytn ecosystem.',
```
Comment (baerae-colin) [inline src/routes.tsx]:
```suggestion
          'The Peb converter from Klaytn is a simple and easy-to-use tool for converting between peb, ston, and KLAY. The page housing this tool provides a comprehensive explanation of these different units and their relationship to the expenditure of Gas in the Klaytn ecosystem.',
```

**Comment by baerae-colin** [comment]
Sorry for the late information, but can we utilize the functions in caver-js?
https://docs.klaytn.foundation/content/dapp/sdk/caver-js/api-references/caver.utils#topeb
https://docs.klaytn.foundation/content/dapp/sdk/caver-js/api-references/caver.utils#convertfrompeb

**Comment by Yeonju-Kim** [comment]
> Sorry for the late information, but can we utilize the functions in caver-js? https://docs.klaytn.foundation/content/dapp/sdk/caver-js/api-references/caver.utils#topeb https://docs.klaytn.foundation/content/dapp/sdk/caver-js/api-references/caver.utils#convertfrompeb

@Sotatek-QuangDo 
Online toolkit is for providing caver-js examples as well as webpages for some features like unit converters. As Colin mentioned, can you replace the getValue function with a function using caver-js?

And like the other pages, it would be nice to add a code example at the bottom of the page as a toggle.

**Comment by edwin082** [comment]
Since Quangdo has yet to reply, I am writing an additional comment. How about leaving the extended converter the same and using caver-js for simple unit conversions?

**Comment by Yeonju-Kim** [comment]
> Since Quangdo has yet to reply, I am writing an additional comment. How about leaving the extended converter the same and using caver-js for simple unit conversions?

@edwin082  Here is the link of the recent discussion on Jira. 
https://klaytn-foundation.atlassian.net/browse/KFWEB-17?focusedCommentId=17240&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-17240

File: src/routes.tsx (hunk 1 above)
Comment (Sotatek-QuangDo) [reply src/routes.tsx]: Yes I changed and updated the commit

**Review by Yeonju-Kim — APPROVED** [review]

**Pull request merged**